### DISCUSSION TITLE
Support Multi-Room Playback

### DISCUSSION THREAD

**Discussion by ProfMoo** [jeffvli/feishin#351] [Ideas]
One of the most convenient features about music players such as Roon and Sonos is the ability to manage and play to multiple endpoints. The applications do a nice job of auto-finding endpoints that could receive playback, both connected to the server and the client.

This feature could enable playing to multiple streamers and/or speakers on the LAN. For me in particular, I'd like for my girlfriend to be able to play different music in the living room than what I have playing in the office.

I've posted an image below of what this looks like in Roon:

<img width="368" alt="Screen Shot 2023-11-01 at 7 28 24 PM" src="https://github.com/jeffvli/feishin/assets/32044819/a21c9e0c-a099-4646-ac29-a4a0c35c1349">

**Comment by ProfMoo** [comment]
It seems there is an open source project called [Music Assistant ](https://github.com/music-assistant/server)that could handle a lot of the heavy lifting on this idea. Feishin would just need to integrate with Music Assistant.

**Reply by lazarevski** [reply]
This would be perfect.
